packages/image_picker/image_picker/example/ios/RunnerUITests/ImagePickerFromLimitedGalleryUITests.m

```diff
@@ -60,7 +47,11 @@ - (void)tearDown {
 }
 
 - (void)testSelectingFromGallery {
-  [self launchPickerAndSelect];
+  if (@available(iOS 14, *)) {
```

Comment:
Please add a comment as to why this is filtered by OS version; it's non-obvious in the immediate code context.